src/main/java/com/google/cloud/resourcemanager/OrgPolicy.java

```diff
@@ -0,0 +1,140 @@
+
+import com.google.api.services.cloudresourcemanager.model.RestoreDefault;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.util.Objects;
+
+/**
+ * A Google Cloud Resource Manager organization policy object.
+ *
+ * <p>Defines a Cloud Organization Policy which is used to specify Constraints for configurations of
+ * Cloud Platform resources.
+ */
+public class OrgPolicy extends OrgPolicyInfo {
+
+  private static final long serialVersionUID = 6767630161335155133L;
+
+  private final ResourceManagerOptions options;
+  private transient ResourceManager resourceManager;
+
+  /** Builder for {@code OrgPolicy}. */
+  public static class Builder extends OrgPolicyInfo.Builder {
+    private final ResourceManager resourceManager;
+    private final OrgPolicyInfo.BuilderImpl infoBuilder;
+
+    Builder(OrgPolicy orgPolicy) {
+      this.resourceManager = orgPolicy.resourceManager;
+      this.infoBuilder = new OrgPolicyInfo.BuilderImpl(orgPolicy);
+    }
+
+    @Override
+    public Builder setBooleanPolicy(BooleanPolicy booleanPolicy) {
+      infoBuilder.setBooleanPolicy(booleanPolicy);
+      return this;
+    }
+
+    @Override
+    public Builder setConstraint(String constraint) {
+      infoBuilder.setConstraint(constraint);
+      return this;
+    }
+
+    @Override
+    public Builder setEtag(String etag) {
+      infoBuilder.setEtag(etag);
+      return this;
+    }
+
+    @Override
+    public Builder setListPolicy(ListPolicy listPolicy) {
+      infoBuilder.setListPolicy(listPolicy);
+      return this;
+    }
+
+    @Override
+    public Builder setRestoreDefault(RestoreDefault restoreDefault) {
+      infoBuilder.setRestoreDefault(restoreDefault);
+      return this;
+    }
+
+    @Override
+    public Builder setUpdateTime(String updateTime) {
+      infoBuilder.setUpdateTime(updateTime);
+      return this;
+    }
+
+    @Override
+    public Builder setVersion(Integer version) {
+      infoBuilder.setVersion(version);
+      return this;
+    }
+
+    @Override
+    public OrgPolicyInfo build() {
+      return new OrgPolicy(resourceManager, infoBuilder);
+    }
+  }
+
+  OrgPolicy(ResourceManager resourceManager, OrgPolicyInfo.BuilderImpl infoBuilder) {
+    super(infoBuilder);
+    this.resourceManager = checkNotNull(resourceManager);
+    this.options = resourceManager.getOptions();
+  }
+
+  /** Returns the {@link ResourceManager} service object associated with this OrgPolicy. */
+  public ResourceManager getResourceManager() {
+    return resourceManager;
+  }
+
+  @Override
+  public Builder toBuilder() {
+    return new OrgPolicy.Builder(this);
+  }
+
+  @Override
+  public final boolean equals(Object obj) {
+    if (obj == this) {
+      return true;
+    }
+    if (obj == null || !obj.getClass().equals(OrgPolicy.class)) {
+      return false;
+    }
+    OrgPolicy other = (OrgPolicy) obj;
+    return Objects.equals(toPb(), other.toPb()) && Objects.equals(options, other.options);
+  }
```

Comment:
Please be consistent, The implementation is different in previous and next classes.